main.lsf

```diff
@@ -155,7 +155,7 @@ Static main
 	Object src/map_events.o
 	Object src/save_local_field_data.o
 	Object asm/unk_0203BA5C.o
-	Object src/diamond_dust.o
+	Object src/field_system_weather.o
```

Comment:
I'm ok with this so long as we're sure this is the only file related to field weather (ie it'd make more sense to use the field_system_weather name for the file that handles the gfx of weather in the field)

Reply:
if you have a better name suggestion, then I can take it on board

Reply:
lets hold off till we decomp more of the weather related files and see if it still works or not? otherwise this literally only affects diamond dust and I'm already struggling with conflicting names 1/3 of the way through battle code so gonna try to avoid that in the field

Reply:
I'd rather not name it diamond_dust.c as the equivalent file in diamond deals with more than just diamond dust

Reply:
The main thing Im concerned about is how much different weather is handled between the two games, since a lot of it was stripped in HGSS